src/loaders.js

```diff
@@ -1,6 +1,10 @@
-if(typeof window === 'undefined' || window !== this) {
-    module.exports = require('./node-loaders');
-}
-else {
-    module.exports = require('./web-loaders');
-}
+'use strict';
+
+(function() {
+    if(typeof window === 'undefined' || window !== this) {
```

Comment:
The comparison doesn't make sense anymore, you've just made `this` always undefined in strict mode. I would ignore this file for linting.